main.c

```diff
@@ -23,7 +23,7 @@
 #include "profiles.h"
 #include "miniFastLED.h"
 
-static void columnCallback(GPTDriver* driver);
+static void mainLoop(GPTDriver* driver);
```

Comment:
I suggest we don't call this mainLoop. Since it is technically not a loop but rather something that runs periodically.

Reply:
Sure, I renamed to `mainCallback`